go-sdk/README.md

```diff
@@ -17,90 +17,107 @@
  under the License.
  -->
 
-# 🚧 Apache Airflow Go Task SDK 🚧
+# Apache Airflow Go Task SDK
+
+The Go SDK uses the Task Execution Interface (TEI or Task API) introduced in AIP-72 with Airflow 3.0.0 to give
+Task functions written in Go full access to the Airflow "model", natively in go.
+
+The Task API however does not provide a means to get the `ExecuteTaskWorkload` to the go worker itself. For
+that we use the Edge Executor API.
+Longer term we will likely need to stabilize the Edge Executor API and add versioning to it.
+
+Since Go is a compiled language (putting aside projects such as [YAEGI](https://github.com/traefik/yaegi) that allow go to be interpreted) all tasks must be a) compiled in to the binary, and b) "registered" inside the worker process in order to be executed.
```

Comment:
```suggestion
Since Go is a **compiled language** (putting aside projects such as [YAEGI](https://github.com/traefik/yaegi) that allow Go to be interpreted), all tasks must:

1. Be compiled into the binary, and  
2. Be registered inside the worker process in order to be executed.
```